docs_app/content/tutorial/1001-but-rxjs-is-hard.md

```diff
@@ -0,0 +1,108 @@
+
+Consider a simple component that will send an HTTP request when you click a button and display the results in a list. You *could* use [`switchMap`](API) and [`map`](API) and [`takeUntil`](API) and a bunch of other operators to get this done... *OR* you could just subscribe to the observable and manage the subscription yourself.
+
+In the simplest case it could look like this:
+
+**React (pseudo-code)**
+
+```ts
+export function SomeComponent() {
+    const subscriptionRef = useRef(null);
+
+    const clickHandler = () => {
+        subscriptionRef.current?.unsubscribe();
+        subscriptionRef.current = ajax.getJSON(url).subscribe(data => {
+            setData(data);
+        })
+    }
+
+    useEffect(() => {
+        return () => {
+            subscriptionRef.current?.unsubscribe();
+        };
+    }, [])
+
+    return <>
+        <button onClick={clickHandler}>Load</button>
+        {data.map((item, i) => <li key={i}>{item}</li>)}
+    </>;
+}
+```
+
+**Angular (pseudo-code)**
+
+NOTE: In angular you may want to use the [async pipe](LINK_HERE).
+
+```ts
+@Component({
+    template: `
+        <button (click)="onClick()">Load</button>
+        <ul><li *ngFor="let item of data">{{item}}<li></ul>
+    `
+})
+const SomeComponent {
+    data: string[] = null;
+
+    constructor(private http: HttpClient) {
+    }
+
+    onClick() {
+        this.subscription?.unsubscribe();
+        this.subscription = this.http.get(url).subscribe(data => this.updateList(data));
+    }
+
+    updateList(data) {
+        this.data = data;
+    }
+
+    ngOnDestroy() {
+        this.subscription?.unsubscribe();
+    }
+}
+```
+
+**Plain DOM Manipulation**
+
+Presented for those who don't quite follow the Angular or React examples above.
+```ts
+let subscription = null;
+
+const list = document.createElement('ul');
+
+const button = document.createElement('button');
+button.textContent = 'Load';
+button.onclick = () => {
+    subscription?.unsubscribe();
+    subscription = ajax.getJSON(url).subscribe(
+        data => {
+            list.innerHTML = data.map(value => `<li>${value}</li>`).join('');
+        }
+    )
+}
+
+document.body.appendChild(button);
+document.body.appendChild(list);
+```
+
+## Solution 2: Experiment And Learn Gradually
+
+In some cases, you've inherited a codebase that is full of RxJS, and you're doomed into needing to maintain it, and you don't quite understand the operators. Maybe our documentation isn't really resonating with you, or maybe you're still uncertain of exactly what a specific operator does.
+
+If this is the case, I'd recommend using an online playground/sandbox app like [Stackblitz](https://stackblitz.com), or [Codesandbox](https://codesandbox.io), et al, and importing `rxjs` and just playing with it.
+
+### Recommendations for playing with operators:
+
```

Comment:
Links to a few sample stackblitz examples could be useful here, giving readers a starting point.